vsintegration/src/FSharp.Editor/FSharp.Editor.fsproj

```diff
@@ -20,7 +21,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <EmbeddedResource Update="FSharp.Editor.resx">
+    <EmbeddedResource Include="FSharp.Editor.resx">
```

Comment:
@brettfo I'm pretty sure this is normal, I think you were only using Update to avoid EnableDefaultEmbeddedResourceItems?

Reply:
Correct, I was using `Update` because `$(EnableDefault...)` was true.  What's the benefit to specifying it this way, e.g., to not allowing the `**/*.resx` glob from running?  The default behavior for SDK projects is to auto-include all `*.resx` files into the `@(EmbeddedResource)` item collection.  I was trying to stay in line with that so that if another `*.resx` file is added to this directory that it's automatically included in the group so we don't have to manually manage the files in the project.